src/worker.cpp

```diff
@@ -194,8 +194,8 @@ bool Worker::isSystemVPDOnDBus() const
     std::array<const char*, 1> interfaces = {
         "xyz.openbmc_project.Inventory.Item.Board.Motherboard"};
 
-    types::MapperGetObject objectMap =
-        dbusUtility::getObjectMap(PIM_PATH_PREFIX + mboardPath, interfaces);
+    types::MapperGetObject objectMap = dbusUtility::getObjectMap(mboardPath,
+                                                                 interfaces);
 
     if (objectMap.empty())
```

Comment:
Suggestion: Not related to the changes in this commit, but the return check block can be simplified to `return !object.empty()`

Reply:
@souvik1914581 If so, please create an issue for this and mention the issue number here. As this is out of scope for this PR, we will take it separately.